monkestation/code/modules/blueshift/items/ash_walker.dm

```diff
@@ -61,6 +61,10 @@
 	icon_state = "mesh"
 	var/list/static/seeds_blacklist = list(
 		/obj/item/seeds/lavaland,
+		/obj/item/seeds/gatfruit,
+		/obj/item/seeds/tree/money,
+		/obj/item/seeds/random,
+		/obj/item/seeds/seedling,
```

Comment:
```suggestion
		/obj/item/seeds/seedling/evil,
```

You prolly want these. This list does **not** include subtypes, and the non-evil seedling seeds appear in the cargo exotic seeds crate + some maps anyways.